### DISCUSSION TITLE
Developer Communication

### DISCUSSION THREAD

**Discussion by kylegrymonprez** [xLightsSequencer/xLights#4940] [General]
@dkulp @computergeek1507 @keithsw1111 is there a developer Slack/Discord out there that some of the more active developers like @derwin12 , @cybercop23 and myself can use to chat with each other, but also you three in more real time to bounce ideas/thoughts and tasks out to? I cant speak with the others, but for me, I know I would benefit from a little more direct communication with you all to not only assist, but also get into some more deeper tasks and ideation going into the next year's development.

Thanks!

**Comment by munroAllie** [comment]
I second this question: @dkulp @computergeek1507 @keithsw1111 . I am a web developer and would love to connect with the main contributors to help build my development environment and sink my teeth into the codebase to help the community.

**Comment by cybercop23** [comment]
https://youtu.be/SWVUQESxnOg

**Comment by dkulp** [comment]
Chatting it over with the FPP devs, we'll likely invite folks to the Falcon Christmas slack space and create an #xLights channel (and possibly others as needed) there.  That's probably the best option as it will allow better communication between the FPP and xLights folks as well.

**Comment by kylegrymonprez** [comment]
that would be awesome! Thanks @dkulp
